xml/System.Reflection.Metadata/MetadataReaderProvider.xml

```diff
@@ -136,12 +139,22 @@
         <Parameter Name="size" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <param name="options">To be added.</param>
-        <param name="size">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="stream">A <see cref="T:System.IO.Stream" /> instance.</param>
+        <param name="options">Options specifying how sections of the image are read from the stream.
```

Comment:
This param description is really long. Probably most of this should be moved to remarks?